src/main/scala/com/lightbend/conductr/sbt/LagomBundlePlugin.scala

```diff
@@ -38,8 +40,9 @@ object LagomPlayBundlePlugin extends AutoPlugin {
       BundleKeys.nrOfCpus := PlayBundleKeyDefaults.nrOfCpus,
       BundleKeys.memory := PlayBundleKeyDefaults.memory,
       BundleKeys.diskSpace := PlayBundleKeyDefaults.diskSpace,
-      conductrBundleLibVersion := Version.conductrBundleLib,
-      libraryDependencies += Library.lagomConductrBundleLib(LagomVersion.current, scalaBinaryVersion.value, conductrBundleLibVersion.value),
+      BundleKeys.endpoints := Map(name.value -> Endpoint("http", 0, Set(URI("http://:9000")))),
```

Comment:
This should check the `enableAcls` flag to determine whether it should be expressed as an ACL or service URI.